README.md

```diff
@@ -240,4 +253,3 @@ You can use the `include_expression` `Filter="PASS"` to achieve this.
 ## Other Resources
 - dockerfiles: https://github.com/d3b-center/bixtools
 
-![pipeline flowchart](./docs/kfdrc-somatic-variant-workflow.cwl.png)
```

Comment:
Perhaps we should delete these PNGs from the repo as well

Reply:
Also this leaves a trailing empty line. That's why you end up with a `|+` in the doc.